### DISCUSSION TITLE
Add blockchain to ediurls.

### DISCUSSION THREAD

**Discussion by ashleney** [Eludris/eludris#81] [Ideas]
What if eldiurs but... good?

**Comment by EnokiUN** [comment]
ok pr it in

**Reply by ashleney** [reply]
no way, keep yourself safe

**Reply by teaishealthy** [reply]
nah this isn't signal or something

**Reply by EnokiUN** [reply]
> nah this isn't signal or something

it is **E**lectronic **DI**ctated **URLS**

**Reply by ashleney** [reply]
I'd PR but idk any rust. Can someone pls do it for me? (I will pay in NFTs)

**Reply by EnokiUN** [reply]
learn it and see

**Comment by ooliver1** [comment]
That's so 2021.
